lib/attentive_sidekiq/manager.rb

```diff
@@ -24,6 +24,10 @@ def update_disappeared_jobs
       # We need to get the new suspicious list again, and remove any lost jobs that are no longer there.
       # Those jobs that appeared in the first suspicious list, but not the second one were simply finished
       # quickly by Sidekiq before showing up as active by a worker.
+
+      # give sidekiq time to update suspicious list, maybe the jobs finished,
+      # but it takes some time until it is removed from the suspicious list
+      sleep(10)
```

Comment:
As far as I understand this method should be executed within `AttentiveSidekiq.timeout_interval` timeframe. Probably something like will work?:
```
    def update_disappeared_jobs
      start_time = Time.now
    
      suspicious  = AttentiveSidekiq::Suspicious.jobs
      active_ids  = AttentiveSidekiq::Active.job_ids
      those_lost  = suspicious.delete_if{|i| active_ids.include?(i["jid"])}

      # give sidekiq time to update suspicious list, maybe the jobs finished,
      # but it takes some time until it is removed from the suspicious list
      wait_time = options[:timeout_interval] - (Time.now - start_time) * 4
      sleep(wait_time) if wait_time > 0
      
      # Sidekiq might have been too fast finishing up a job that appeared in the suspicious list
      # but didn't make it to the active list, so that's a false-positive.
      # We need to get the new suspicious list again, and remove any lost jobs that are no longer there.
      # Those jobs that appeared in the first suspicious list, but not the second one were simply finished
      # quickly by Sidekiq before showing up as active by a worker.
      suspicious  = AttentiveSidekiq::Suspicious.jobs
      those_lost.delete_if{|i| !suspicious.any?{|j| i['jid'] == j['jid']} }

      those_lost.each do |job|
        Disappeared.add(job)
        Suspicious.remove(job['jid'])
      end
    end
```
4x multiplier here to based on assumption that code after `sleep` will execute no longer than 3 times slower than code before `sleep`.